test/modules/data-integrity-service.test.js

```diff
@@ -0,0 +1,43 @@
+const { describe, before, it } = require('mocha');
+const { assert, expect } = require('chai');
+const DataIntegrityResolver = require('../../modules/service/data-integrity/data-integrity-resolver');
+
+const signingWallet = {
+    wallet: '0x324b939670d154667466b17524d9136d879CDC09',
+    privateKey: '1556ba8faf6c2a1e696e4a70a0b7e1c6582ba26b9d8e6a9a7b96b22a29a5d2d3',
+};
+
+const message = 'message for signing';
+
+describe('Data integrity service', () => {
```

Comment:
I'd just recommend adding a test that an invalid signature recovers a different address and fails the validation. Maybe we create a signature then flip a bit